drghs/v1/service_resources.pb.go

```diff
@@ -1,17 +1,3 @@
-// Copyright 2019 Google LLC
```

Comment:
Don't you need these copyright headers?

Reply:
Good catch! The `protoc` code generator removes the headers every time.